client/src/components/layout/NewReviewForm.js

```diff
@@ -0,0 +1,85 @@
+import React, { useState } from "react"
+import { useParams, Redirect } from "react-router-dom"
+import translateServerErrors from "../../services/translateServerErrors"
+import ErrorList from "./ErrorList"
+
+const NewReviewForm = (props) => {
+    const { id } = useParams()
+    
+    const [newReview, setNewReview] = useState({
+        userId: props.user.id,
```

Comment:
the front end should not be tracking the user creating the form. the back end is holding and managing the current user via `req.user`. Form state should only hold onto information supplied by the user - the user is not filling out in the form their user id or the destination id. Add these values in the back end before inserting the record.